src/target/stm32f1.c

```diff
@@ -272,13 +276,16 @@ void mm32l0_mem_write_sized(adiv5_access_port_s *ap, uint32_t dest, const void *
 		case ALIGN_BYTE: {
 			uint8_t value;
 			memcpy(&value, src, sizeof(value));
-			tmp = (uint32_t)value << (8U * (dest & 3U));
+			tmp = (uint32_t)value;
+			tmp = tmp | tmp << 8U;
+			tmp = tmp | tmp << 16U;
 			break;
 		}
 		case ALIGN_HALFWORD: {
 			uint16_t value;
 			memcpy(&value, src, sizeof(value));
-			tmp = (uint32_t)value; /* MM32 special, no shift << (8U * (dest & 2U)) */
+			tmp = (uint32_t)value;
+			tmp = tmp | tmp << 16U;
```

Comment:
Similar to above - comment such as "Copy the value to writte to both halves of the uint32_t" would be useful

Reply:
ok, done.